packages/nx-playwright/src/generators/project/root-config/playwright.config.base.ts__tmpl__

```diff
@@ -2,18 +2,10 @@ import { PlaywrightTestConfig } from '@playwright/test';
 
 const baseURL = process.env.E2E_BASE_URL || 'http://localhost:4200/';
 
-const desktop = { width: 1200, height: 2000 };
-
 export const baseConfig: PlaywrightTestConfig = {
   retries: 3,
   maxFailures: 2,
   timeout: 120000,
-  projects: [
```

Comment:
this conflicts with `browser` param an might be considered over-opinionated for an initial setup.  your choice of course :)

Reply:
these are leftovers from our initial implementation so we are happy with this change